### PR TITLE
Rewriting blurb

### PR DESCRIPTION
**Pull request by gvwilson**
More words on the home page.

### CODE REVIEW

**Comment by gvwilson** [comment]
Fixes #132.

File: index.md [hunk 1]
```diff
@@ -2,13 +2,35 @@
...
+
+This two-day class will introduce participants to a handful of key research findings
+and show how they can be used to help people learn better and faster.
+We will look at the cognitive differences between novices, competent practitioners, and experts;
+why those differences require different pedagogical approaches;
+how to design formative assessments to provide feedback to both instructors and students while learning is taking place;
+how to mitigate cognitive load during learning;
+how concept maps can be used both to design lessons and to communicate those designs;
+how practices similar to pair programming and test-driven design can be used in education;
+and at teaching as a performance art.
```
Comment (arokem) [inline index.md]: Is this "and about teaching as a performance art"?

**Comment by arokem** [comment]
:+1:

File: index.md (hunk 1 above)
Comment (gvwilson) [inline index.md]: Good catch.

**Comment by arokem** [comment]
I will let others provide feedback here, before merging.

**Pull request merged**